tutorials/vo/conesearch_tutorial.ipynb

```diff
@@ -0,0 +1,438 @@
+     "cell_type": "markdown",
+     "metadata": {},
+     "source": [
+      "It might be useful to list the available Cone Search catalogs first. By default, catalogs that pass nightly validation are included. Validation is hosted by Space Telescope Science Institute (STScI)."
+     ]
+    },
+    {
+     "cell_type": "code",
+     "collapsed": false,
+     "input": [
+      "conesearch.list_catalogs()"
+     ],
+     "language": "python",
+     "metadata": {},
+     "outputs": []
+    },
+    {
+     "cell_type": "markdown",
+     "metadata": {},
+     "source": [
+      "Next, let's pick an astronomical object of interest. For example, M31."
+     ]
+    },
+    {
+     "cell_type": "code",
+     "collapsed": false,
+     "input": [
+      "from astropy import coordinates as coord\n",
+      "\n",
+      "c = coord.SkyCoord.from_name('M31', frame='icrs')\n",
+      "print(c)"
+     ],
+     "language": "python",
+     "metadata": {},
+     "outputs": []
+    },
+    {
+     "cell_type": "markdown",
+     "metadata": {},
+     "source": [
+      "By default, a basic Cone Search goes through the list of catalogs and *stops* at the first one that returns non-empty VO table. Let's search for objects within 0.1 degree around M31. You will see a lot of warnings that were generated by VO table parser but ignored by Cone Search service validator. VO compliance enforced by Cone Search providers is beyond the control of `astropy.vo` package."
+     ]
+    },
+    {
+     "cell_type": "code",
+     "collapsed": false,
+     "input": [
+      "from astropy import units as u\n",
+      "\n",
+      "result = conesearch.conesearch(c, 0.1 * u.degree)"
+     ],
+     "language": "python",
+     "metadata": {},
+     "outputs": []
+    },
+    {
+     "cell_type": "code",
+     "collapsed": false,
+     "input": [
+      "print('First non-empty table returned by', result.url)\n",
+      "print('Number of rows is', result.nrows)"
+     ],
+     "language": "python",
+     "metadata": {},
+     "outputs": []
+    },
+    {
+     "cell_type": "markdown",
+     "metadata": {},
+     "source": [
+      "Extract metadata of this Cone Search catalog."
+     ]
+    },
+    {
+     "cell_type": "code",
+     "collapsed": false,
+     "input": [
+      "from astropy.vo.client import vos_catalog\n",
+      "\n",
+      "my_db = vos_catalog.get_remote_catalog_db(conesearch.CONESEARCH_DBNAME())\n",
+      "my_cat = my_db.get_catalog_by_url(result.url)\n",
+      "print(my_cat.dumps())"
+     ],
+     "language": "python",
+     "metadata": {},
+     "outputs": []
+    },
+    {
+     "cell_type": "markdown",
+     "metadata": {},
+     "source": [
+      "If you have a favorite catalog in mind, you can also perform Cone Search only on that catalog."
+     ]
+    },
+    {
+     "cell_type": "code",
+     "collapsed": false,
+     "input": [
+      "result = conesearch.conesearch(\n",
+      "    c, 0.1 * u.degree, catalog_db='The USNO-A2.0 Catalogue (Monet+ 1998) 1')"
```

Comment:
Could you describe or link to documentation here that specifies what are valid inputs for `catalog_db` (e.g., where can I find the valid catalog names)